skrub/_datetime_encoder.py

```diff
@@ -125,7 +124,7 @@ def _more_tags(self):
         }
 
     def _validate_keywords(self):
-        if self.extract_until not in TIME_LEVELS:
+        if self.extract_until not in TIME_LEVELS and not self.extract_until is None:
```

Comment:
it's an unimportant style detail but most people would prefer `x is not None` to `not x is None`